components/calculator.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [answer, setAnswer] = useState('')
+  const [error, setError] = useState('')
+  const [numOne, setNumOne] = useState('')
+  const [numTwo, setNumTwo] = useState('')
+  const [operator, setOperator] = useState('')
+
+  const calculateAndUpdateAnswer = () => {
+    const num1 = Number(numOne)
+    const num2 = Number(numTwo)
+    if (num1 && num2) {
+      if (error) setError('')
+      switch (operator) {
+        case 'add':
+          setAnswer(num1 + num2)
+          break
+        case 'subtract':
+          setAnswer(num1 - num2)
+          break
+        case 'multiply':
+          setAnswer(num1 * num2)
+          break
+        case 'divide':
+          setAnswer(num1 / num2)
+          break
+        default:
+          setAnswer(num1 + num2)
```

Comment:
This should be an error state. Defaulting to addition feels weird on the front end.